### ISSUE TITLE
Sound Card

### ISSUE THREAD

**Issue by eMerzh** [h16nning/skola#52]
It's probably not a priority... even for me :p 

but I stumble upon this https://jessicalieb.itch.io/birdlingo ... and thought hey, but this (is nice) should be a flashcard deck 🤔 ...

and it seems anki as one 😲 ... 
so why not have some sound support... this could be "easy" since we have attachments already ...

**Comment by h16nning** [comment]
That is certainly useful! Would you make a new CardType it? Think we could just add the option to upload audio from sound file (maybe we need a custom tiptap extension for that, don't know) and add some styling.

That playing sounds could be used in normal cards, double-sided cards etc.

**Comment by eMerzh** [comment]
yeah new type is obviously not needed ... agree it can be usefull both normal & double sided... 👍
